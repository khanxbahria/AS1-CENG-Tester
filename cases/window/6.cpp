#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->isEmpty();
Tab E = Tab("kkk", "y","qqq");
wind->moveActiveTabTo(5);
Tab K = Tab("u", "rrr","sss");
wind->changeActiveTabTo(1);
Tab M = Tab("fff", "rr","yyy");
wind->changeActiveTabTo(3);
wind->print();
Tab Q = Tab("xx", "uuu","cc");
wind->moveActiveTabTo(0);
Tab L = Tab("ggg", "o","j");
wind->getActiveTab();
wind->print();
wind->isEmpty();
Tab Y = Tab("fff", "kk","yyy");
wind->getActiveTab();
Tab N = Tab("rr", "tt","iii");
wind->moveActiveTabTo(0);
wind->closeTab();
Tab F = Tab("oo", "eee","iii");
wind->getActiveTab();
Tab S = Tab("zz", "oo","q");
wind->changeActiveTabTo(4);
Tab P = Tab("kkk", "nnn","tt");
wind->getActiveTab();
Tab U = Tab("w", "uu","n");
wind->moveActiveTabTo(5);
Tab T = Tab("bbb", "g","j");
wind->moveActiveTabTo(5);
wind->moveActiveTabTo(0);
wind->closeTab();
wind->isEmpty();
Tab B = Tab("ddd", "ddd","xx");
wind->newTab(B);
wind->closeTab();
Tab C = Tab("oo", "g","ddd");
wind->changeActiveTabTo(0);
Tab J = Tab("sss", "rrr","j");
wind->closeTab();
Tab Z = Tab("r", "b","uu");
wind->isEmpty();
Tab W = Tab("ff", "ss","nn");
wind->newTab(C);
wind->changeActiveTabTo(4);
wind->closeTab();
wind->closeTab();
wind->changeActiveTabTo(5);
wind->isEmpty();
wind->isEmpty();
wind->changeActiveTabTo(1);
Tab D = Tab("bb", "r","qq");
wind->print();
wind->changeActiveTabTo(1);
wind->closeTab();
wind->newTab(D);
wind->isEmpty();
wind->print();
Tab V = Tab("uu", "rr","ttt");
wind->isEmpty();
Tab H = Tab("k", "iii","kk");
wind->changeActiveTabTo(1);
wind->changeActiveTabTo(3);
wind->changeActiveTabTo(2);
wind->newTab(E);
Tab I = Tab("ttt", "yyy","ww");
wind->getActiveTab();
wind->isEmpty();
wind->isEmpty();
Tab G = Tab("yyy", "ttt","o");
wind->newTab(F);
wind->isEmpty();
wind->changeActiveTabTo(5);
wind->newTab(G);
wind->closeTab();
wind->newTab(H);
wind->moveActiveTabTo(3);
wind->closeTab();
wind->isEmpty();
wind->closeTab();
wind->isEmpty();
wind->changeActiveTabTo(0);
wind->print();
wind->changeActiveTabTo(2);
wind->closeTab();
wind->getActiveTab();
wind->isEmpty();
wind->getActiveTab();
wind->moveActiveTabTo(2);
wind->getActiveTab();
wind->moveActiveTabTo(1);
wind->getActiveTab();
wind->getActiveTab();
wind->print();
wind->closeTab();
wind->newTab(I);
wind->isEmpty();
wind->changeActiveTabTo(5);
wind->newTab(J);
wind->print();
wind->print();
wind->print();
wind->isEmpty();
wind->isEmpty();
wind->newTab(K);
Tab R = Tab("u", "ppp","ggg");
wind->closeTab();
wind->newTab(L);
wind->getActiveTab();
wind->newTab(M);
wind->newTab(N);
wind->closeTab();
wind->changeActiveTabTo(1);
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(2);
Tab A = Tab("yy", "c","yyy");
wind->closeTab();
wind->newTab(A);
wind->moveActiveTabTo(4);
wind->moveActiveTabTo(5);
wind->newTab(P);
wind->closeTab();
wind->newTab(Q);
wind->closeTab();
wind->newTab(R);
delete wind;
}
	return 0;
}