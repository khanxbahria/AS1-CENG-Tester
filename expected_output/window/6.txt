The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 0
r
bb
qq

The active tab is tab 0
r
bb
qq

y
kkk
qqq

The active tab is tab 0
y
kkk
qqq

The active tab is tab 1
yyy
ttt
ww

rrr
sss
j

The active tab is tab 1
yyy
ttt
ww

rrr
sss
j

The active tab is tab 1
yyy
ttt
ww

rrr
sss
j

